androidtv/constants.py

```diff
@@ -564,6 +564,12 @@ class DeviceEnum(IntEnum):
 #: Default authentication timeout (in s) for :meth:`adb_shell.handle.tcp_handle.TcpHandle.connect` and :meth:`adb_shell.handle.tcp_handle_async.TcpHandleAsync.connect`
 DEFAULT_AUTH_TIMEOUT_S = 10.0
 
+#: Default transport timeout (in s) for :meth:`adb_shell.handle.tcp_handle.TcpHandle.connect` and :meth:`adb_shell.handle.tcp_handle_async.TcpHandleAsync.connect`
+DEFAULT_TRANSPORT_TIMEOUT_S = 1.0
+
+#: Maximum allowed transport timeout (in s) for :meth:`adb_shell.handle.tcp_handle.TcpHandle.connect` and :meth:`adb_shell.handle.tcp_handle_async.TcpHandleAsync.connect`
+MAX_TRANSPORT_TIMEOUT_S = 5.0
```

Comment:
I think it's better to not have a max. Whatever the user specifies, just use that value.